lib/rules/web/ocm_console/cluster_detail_monitoring_tab.xyaml

```diff
@@ -3,6 +3,38 @@ click_monitoring_tab:
         selector:
             xpath: //button[contains(.,'Monitoring')]
         op: click
+click_settings_tab:
+    element:
+        selector:
+            xpath: //button[contains(.,'Settings')]
+        op: click
+click_overview_tab:
+    element:
+        selector:
+            xpath: //button[contains(.,'Overview')]
+        op: click
+check_user_monitoring_checkbox_disabled:
+    element:
+        selector:
+            xpath: //input[@disabled]
+check_user_monitoring_checkbox_unchecked:
+    element:
+        selector:
+            xpath: //input[@value='false']
+check_user_monitoring_checkbox_checked:
+    element:
+        selector:
+            xpath: //input[@value='true']
+refresh_detail_page_for_settings_tab:
```

Comment:
The action refresh_detail_page will refresh the page and also **check_osd_details_in_detail_page**, which is not what I want because we are in the settings tab.

Thus, I write a new refresh step for the settings tab as you saw

Reply:
OK